client/changelist/change.go

```diff
@@ -0,0 +1,73 @@
+package changelist
+
+import (
+	"encoding/json"
+
+	"github.com/endophage/gotuf/data"
+)
+
+type tufChange struct {
+	Actn       string `json:"action"`
+	Role       string `json:"role"`
+	ChangeType string `json:"type"`
+	ChangePath string `json:"path"`
+	Data       []byte `json:"data"`
+}
+
+func NewTufChange(action, role, changeType, changePath string, content []byte) *tufChange {
+	return &tufChange{
+		Actn:       action,
+		Role:       role,
+		ChangeType: changeType,
+		ChangePath: changePath,
+		Data:       content,
+	}
+}
+
+func NewTufTargetChange(action, target string, hash map[string]data.HexBytes, size int64, custom map[string]interface{}) (*tufChange, error) {
```

Comment:
Needs comment. Not sure what it's going on here with the custom. Is it a best effort addition of extra params?